doc/userguide/src/CreatingTestData/CreatingUserKeywords.rst

```diff
@@ -407,25 +394,18 @@ text can help but, for example, the test below fails because keyword
 :name:`I execute "ls" with "-lh"` matches both of the defined
 keywords.
 
-.. table:: Embedded arguments match too much
-   :class: example
-
-   ============================  ===============================
-             Test Case                         Step
-   ============================  ===============================
-   Example                       I execute "ls"
-   \                             I execute "ls" with "-lh"
-   ============================  ===============================
+.. sourcecode:: robotframework
 
-.. table::
-   :class: example
+   *** Test Cases ***   
+   Example
+       I execute "ls"
+       I execute "ls" with "-lh"
 
-   =================================  ==========  ==============  ==========
-                Keyword                  Action      Argument      Argument
-   =================================  ==========  ==============  ==========
-   I execute "${cmd}"                 Run         ${cmd}
-   I execute "${cmd}" with "${opts}"  Run         ${cmd} ${opts}
-   =================================  ==========  ==============  ==========
+   *** Keywords ***
+   I execute "${cmd}"
+       Run    ${cmd}
+   I execute "${cmd}" with "${opts}"
+       Run    ${cmd}    ${opts}
```

Comment:
Empty row between keywords. Use `Run Process` instead of `Run` (which isn't even used correctly)?